src/locales/ko/battle.ts

```diff
@@ -14,6 +14,7 @@ export const battle: SimpleTranslationEntries = {
   "switchQuestion": "{{pokemonName}}[[를]]\n교체하시겠습니까?",
   "trainerDefeated": "{{trainerName}}[[와]]의\n승부에서 이겼다!",
   "moneyWon": "상금으로\n₽{{moneyAmount}}을 손에 넣었다!",
+  "moneyPickedUp": "You picked up ₽{{moneyAmount}}!",
```

Comment:
```suggestion
  "moneyPickedUp": "₽{{moneyAmount}}을 주웠다!",
```
![image](https://github.com/pagefaultgames/pokerogue/assets/47556641/b9e0dfa8-447b-4c05-af1b-7c00d17f4276)

I suggest a Korean translation as above.

Reply:
@KimJeongSun 
Minor thing but could you replace '을' to [[를]] ?

```suggestion
  "moneyPickedUp": "₽{{moneyAmount}}[[를]] 주웠다!",
```